docker-compose.yml

```diff
@@ -18,14 +23,14 @@ services:
     image: orchestrator_image
     restart: always
     environment:
-      DAGSTER_POSTGRES_USER: "postgres_user"
-      DAGSTER_POSTGRES_PASSWORD: "postgres_password"
-      DAGSTER_POSTGRES_DB: "postgres_db"
+      DAGSTER_POSTGRES_USER: ${POSTGRES_USER}
```

Comment:
you can use yaml balise here based on the `x-orchestrator-config` too

Reply:
Should be good now